src/utils/splitNSearch.ts

```diff
@@ -0,0 +1,4 @@
+export const splitNSearch = (query: string) => {
+    const searchValues = query.split(',');
+    console.log('Searching', searchValues);
```

Comment:
remove this

Reply:
Since the feature is behind feature flag and currently the functionality is missing, thus I have used console log. Once I make the api integration the console log will beremoved